packages/languages/id/src/lastnames.json

```diff
@@ -0,0 +1 @@
","wailan","wailayati","waiman","waimin","wainah","wainu","waisin","waito","wajahtu","wajdi","wajid","wajiman","wajis","wajito","wajiyana","wajong","wakartimah","watim","watimah","waliman","watimiri","walimom","watinturan","walirimba","watito","waliuah","waliyem","waliyul","waljimin","wajjito","waljoto","walkiah","walkiatmo","watman","walmen","walmon","walmudi","watni","watoejo","waloeyo","watonguru","walukow","waluku","waluya","watuyadi","waluyanti","watuyanty","waluyo","watuyono","watyasih","watyati","wamah","waman","wamat","wamin","wamit","wamoha","wanari","wandari","wandi","wandono","wandoyo","waneng","wanfik","wangkuto","wangsing","wangso","waniati","wanif","wanih","waning","wanisa","wanito","waniwati","wanoyo","wantawirdja","wanti","wantini","wanto","wantoro","wantoyo","warantinah","warastuti","warbini","wardana","wardanas","wardani","wardaya","warde","wardhama","wardiah","wardian","wardiana","wardianto","wardiayati","wardidjah","wardija","wardijono","wardiman","wardimin","wardin","wardingningwan","wardiningsih","wardiono","wardius","wardiyah","wardiyani","wardiyanto","wardiyat","wardiyo","wardjito","wardjoko","wardju","wardojo","wardoyo","wardoyono","wargiya","wargo","wargono","warhadi","warhati","warheani","warid","warida","waridah","waridan","warddi","waridin","waridjan","waridjo","warijem","warijo","warim","wariman","warimpi","warindi","warindra","waringatun","warini","warino","warip","waris","warisa","warisiyanto","warisman","warismun","warisni","warisno","wibisono","wiyoto","wiyuthana","wizyarni","woersito","woeryanto","wohargo","wondokop","woro","wowiek","woyono","wudy","wuhardjono","wutan","wulanda","wutandary","wulandati","wulani","wulansari","wutantari","wurdia","wuda","yanti","yatinus","yarman","yasah","yasan","yasin","yasmon","yasmudi","yasmun","yasnani","yasni","yasniarti","yasniati","yasnimar","yasnina","yasniwarti","yasnizar","yasnut","yaspen","yasri","yasrianto","yastoyo","yastuti","yasuko","yaswar","yaswin","yata","yatim","yatimah","yatimbasid","yatimin","yatin","yatinah","yatini","yatiran","yatirun","yatman","yatmi","yatmiadi","yatmo","yatono","yatt","yatti","yatto","yayan","yayanto","yayasatia","yayat","yayi","yayin","yayo","yayong","yayu","yazid","yedat","yedismen","yekti","yetno","yeyep","yiyas","yoedi","yoewono","yomo","yomun","yona","yonara","yono","yospi","yoyok","yozin","yuda","yudi","yudika","yuhadi","yuli","yulis","yumardi","yunarso","zaenat","zahara","zahir","zahroni","zaianima","zaianona","zaifut","zainat","zainaldi","zainar","zainawi","zainedin","zaini","zainibor","zainih","zainimar","zainir","zainuar","zainudin","zainut","zairawati","zairi","zairia","zairion","zairman","zairo","zairut","zairumeh","zairus","zaiwat","zaizar","zajadi","zajib","zakarja","zaki","zalnebar","zamal","zamawi","zambari","zamboud","zamhari","zamri","zamrodin","zamroni","zamrudy","zamsusry","zamuddin","zamzam","zanizami","zamzani","zandra","zantoro","zarti","zayyanah","ziyanti","zoebir","zubaedi","zulfah","zulfikar","zutifrani","zwarman"]
```

Comment:
firstname.json has the same name is this correct?

Reply:
Yes, people in Indonesia rarely use surnames, so first and last names can be reversed. Do I need to include only `names.json`?

I get this list of names from [Wikipedia](https://en.wiktionary.org/wiki/Appendix:Indonesian_given_names).

Reply:
Yes `names.json` is enough if its the same data anyway.

If you have it from wikipedia you could use the `HTMLGenerator` like for the `pt-b` lastnames dictionary. 
You just need to adjust the extractorFunction a little bit

Reply:
I plan to add more names from several sources from blogs and articles, so I made [my own gist list](https://gist.github.com/bagaswastu/35c80cdd8c32ca0b6aafa119e80e93ef). Is that a bad idea?

Reply:
Then feel free to use your own gist list. But it would be nice if you add your sources to the readme :)

Reply:
Sure thing! I'll add the source to the README.md on the `@zxcvbn-ts/language-id` package.